applications/tari_app_grpc/proto/base_node.proto

```diff
@@ -80,6 +80,13 @@ message NewBlockTemplateResponse{
     MinerData miner_data = 4;
 }
 
+/// return type of GetNewBlockTemplate
+message NewBlockTemplateRequest{
+    PowAlgo algo = 1;
+    //This field should be moved to optional once optional keyword is standard
```

Comment:
Comment is not needed since all fields are optional in protobuf

Reply:
Fields for messages are optional, but not primitives:
This was released 5 days ago:
https://github.com/protocolbuffers/protobuf/releases/tag/v3.15.0